src/input/mod.rs

```diff
@@ -2578,7 +2641,13 @@ mod tests {
         assert!(matches!(filter, FilterResult::Forward));
 
         let filter = close_key_event(&mut suppressed_keys, mods, false);
-        assert!(matches!(filter, FilterResult::Intercept(None)));
+        assert!(matches!(
+            filter,
+            FilterResult::Intercept(Some(Bind {
+                action: Action::CloseWindow,
+                ..
+            }))
+        ));
 
         let filter = none_key_event(&mut suppressed_keys, mods, false);
         assert!(matches!(filter, FilterResult::Forward));
```

Comment:
I can't comment on the lines below, but you can see in the test right below this line that if you press Mod then Q then release Mod, you won't get a release event with the bind, so the key repeat will go on forever.

Reply:
resolved